app/assets/javascript/app/components/MapLabel.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import Hourglass from './Hourglass';
+
+class MapLabel extends React.Component {
+
+  render() {
+    const icon = (() => {
+      if (this.props.type === 'plan') {
+        let hourglassType = 'empty';
+        const dataYear = parseInt(this.props.items[0].top);
+        const currentYear = parseInt((new Date()).getFullYear()); 
+        const diff = Math.abs(dataYear - currentYear);
+
+        if (diff >= 3) {
```

Comment:
These diffs should be constants rather than magic numbers. I know what you mean, but we're probably going to forget at some point.